src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -79,6 +82,8 @@
   @GraphQLInputField
   private String code;
 
+  private ArrayList<HashMap<String, String>> userActivities;
```

Comment:
`ArrayList` and `HashMap` are implementation details; declare this as `List<Map<String, String>>`